CHANGELOG.md

```diff
@@ -12,39 +12,79 @@
 
 ### Minor Changes
```

Comment:
I think the dual step nature of this makes it a major change, but that ship has sailed, right? We can't make this version 62?

Reply:
Hmm personally I'd find it surprising to have to go through this process for just a minor version bump, but if you think in the context of this library it doesn't quite make the cut for a major I'm happy to leave it. I think in the end it doesn't really make much practical difference, given clients tend to update to the latest version anyway.